tests/unit_tests/test_tethys_portal/test_context_processors.py

```diff
@@ -1,24 +1,59 @@
-import unittest
+from unittest import TestCase, mock
 from django.test import override_settings
 from tethys_portal import context_processors
 
 
-class TestStaticDependency(unittest.TestCase):
+class TestTethysPortalContext(TestCase):
     def setUp(self):
         pass
 
     def tearDown(self):
         pass
 
-    @override_settings(MULTIPLE_APP_MODE=False)
-    def test_check_single_app_mode_single(self):
-        single_app_mode, single_app_name = context_processors.check_single_app_mode()
+    def test_context_processors_multiple_app_mode(self):
```

Comment:
Avoid assuming the values of settings. Use the settings.override here to set `MULTIPLE_APP_MODE` to `True` explicitly.

Reply:
sounds good. Ill go through and update those tests